src/plugins/GamepadManager.js

```diff
@@ -351,7 +361,10 @@ module.exports = class GamepadManager {
      * @param {number} remoteIndex index of this gamepad in the remote VM
      */
     listenForInputs(localIndex, remoteIndex) {
-        this.currentGamepads[localIndex] = {remoteIndex: remoteIndex, buttons: []};
+        const gamepad = this.getRawGamepad(remoteIndex);
```

Comment:
it's very weird to me that we use the remoteIndex here?  I think in the raw gamepads the `index` is the local one?